castai/resource_castai_autoscaler_test.go

```diff
@@ -183,6 +183,7 @@ func TestAutoscalerResource_PoliciesUpdateAction(t *testing.T) {
 
 			return &http.Response{
 				StatusCode: 200,
+				Body:       io.NopCloser(bytes.NewReader([]byte(""))),
```

Comment:
I would be good to have a triangulation, right now this test case only asserts with we are successful with the request to PoliciesAPIUpsertClusterPoliciesWithResponse, but we have no unit test for the error scenario

Reply:
Will add an error test case